### DISCUSSION TITLE
PorousFlow - Storage units

### DISCUSSION THREAD

**Discussion by MatiasAllay** [idaholab/moose#18200] [Q&A Modules: General] [answered]
Dear MOOSE,
I got a simple question. In the [GW](https://mooseframework.inl.gov/modules/porous_flow/groundwater_models.html) examples the relation between Specific Storage in Pa^-1 and in m^-1 is simply represented by eq. 11?
Thanks!
Matias

**Comment by WilkAndy** [comment, marked as answer]
Yes, that is correct.  Remember that, although that equation is written with an equals-sign (=) there are various approximations needed to derive it (hence the preceding "To reasonable accuracy").

**Reply by MatiasAllay** [reply]
Thanks Andy!
